bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1825,8 +1829,10 @@ lnd/clm2/surfdata_esmf/ctsm5.3.0/surfdata_mpasa120_hist_1850_78pfts_c240908.nc</
 lnd/clm2/surfdata_esmf/ctsm5.3.0/surfdata_ne30np4_hist_1850_78pfts_c240908.nc</fsurdat>
 <fsurdat hgrid="ne30np4.pg2" sim_year="1850">
 lnd/clm2/surfdata_esmf/ctsm5.3.0/surfdata_ne30np4.pg2_hist_1850_78pfts_c240908.nc</fsurdat>
-<fsurdat hgrid="ne30np4.pg3" sim_year="1850">
+<fsurdat hgrid="ne30np4.pg3" sim_year="1850" cmip_era="cmip7">
 lnd/clm2/surfdata_esmf/ctsm5.4.0/surfdata_ne30np4.pg3_hist_1850_78pfts_c250723.nc</fsurdat>
+<fsurdat hgrid="ne30np4.pg3" sim_year="1850" cmip_era="cmip6">
+lnd/clm2/surfdata_esmf/ctsm5.3.0/surfdata_ne30np4.pg3_hist_1850_78pfts_c240908.nc</fsurdat>
```

Comment:
From meeting with @ekluzek 
Adding cmip7 flag to the same three grids for historical landuse files but not the cmip6 flag. Check how asking for cmip6 fails for such a case.

Reply:
This test `SMS_Ld1.f10_f10_mg37.ISSP245Clm60BgcCropCrujra.casper_intel.clm-default` fails as it should when I change it to cmip7. So far, I have not gotten the namelist test to fail. See test.out4 for latest attempt.